packages/server-wallet/src/wallet/store.ts

```diff
@@ -461,6 +466,124 @@ export class Store {
     return result;
   }
 
+  // FIXME: Turn into SQL tables
+  public eraseLedgerDataFromMemory(): void {
+    this.ledgerRequests = new LedgerRequests();
+    this.ledgers = {};
+  }
```

Comment:
Relevant for #2693 perhaps.  Will also need to replicate work done here to "SQL-ify" these in memory ledger constructs I have made.